.github/workflows/release.yaml

```diff
@@ -3,59 +3,55 @@ name: Release
 on:
   push:
     tags:
-      - "*"
+      - v[0-9]+.*
 jobs:
-  build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, macos-latest]
-        rust: [stable]
+  body:
+    runs-on: ubuntu-latest
     permissions:
       contents: write
-
-    runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@master
-
-      - name: Install just
-        uses: taiki-e/install-action@just
-
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-
-      - name: Setup build env for Linux
-        if: matrix.os == 'ubuntu-latest'
-        run: |
-          sudo apt update
-          sudo apt install -y musl-tools gcc
-          rustup target add x86_64-unknown-linux-musl
-
-      - name: Build for Linux
-        if: matrix.os == 'ubuntu-latest'
-        run: |
-          just build-release-linux
-          cp target/x86_64-unknown-linux-musl/release/tenere tenere-x86_64-linux-musl
-
-      - name: Build for macos
-        if: matrix.os == 'macos-latest'
-        run: |
-          rustup target add aarch64-apple-darwin
-          just build-release-macos
-          cp target/x86_64-apple-darwin/release/tenere tenere-x86_64-macos
-          cp target/aarch64-apple-darwin/release/tenere tenere-aarch64-macos
-
+        uses: actions/checkout@v4
       - name: Extract release notes
-        if: matrix.os == 'ubuntu-latest'
+        uses: ffurrer2/extract-release-notes@v2
         id: release_notes
-        uses: ffurrer2/extract-release-notes@v1
+      - name: Upload Body
+        uses: softprops/action-gh-release@v2
+        with:
+          body: ${{ steps.release_notes.outputs.release_notes }}  
 
-      - name: Release
-        uses: softprops/action-gh-release@v1
+  build:
+    permissions:
+      contents: write
+    continue-on-error: true
+    strategy:
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
```

Comment:
should use `musl` instead of `gnu`

Reply:
I thought about that. 
Do you know of users that don't have glibc?

Reply:
it is not about having glibc. it is about using statically linked binaries. You can add the `gnu` one but do not remove the `musl`

Reply:
Ok